test/helpers/privateMocksSpec.js

```diff
@@ -9,7 +9,7 @@ describe('private mocks', function() {
         spyOn(window, 'it');
 
         they('should do stuff with $prop', ['a', 'b', 'c']);
-        expect(window.it.calls.length).toEqual(3);
```

Comment:
All of these could be converted instead to `.calls.count()).toEqual(...` This would make it similar to what is was before.